### DISCUSSION TITLE
Tabbed Field Groups

### DISCUSSION THREAD

**Discussion by guilhermevrs** [pods-framework/pods#6655] [Ideas]
Hi,

Is there any plans to include the possibility of create tabs to a pod? Similar to ACF (Advanced Custom Fields)..

I was reading your code and I saw there are no hooks that would enable me to extend this exact part (the one that creates the metabox and stuff).

Cheers

**Comment by sc0ttkclark** [comment]
We don't currently offer tabbing functionality within metaboxes, we recommend using Tabify Edit Screen instead as it provides a more consistent experience in the editor for users to get around.
